raiden_contracts/tests/test_endpointregistry.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+from raiden_contracts.utils.events import check_address_registered
+
+
+def test_endpointregistry(endpoint_registry_contract, get_accounts, event_handler):
+    (A, B) = get_accounts(2)
+    ev_handler = event_handler(endpoint_registry_contract)
+    PORT = '127.0.0.1:38647'
+    endpoint_registry_contract.transact({'from': A}).registerEndpoint(PORT)
+    assert endpoint_registry_contract.call().findAddressByEndpoint(PORT) == A
+    NEW_PORT = '192.168.0.1:4002'
+    txn_hash = endpoint_registry_contract.transact({'from': A}).registerEndpoint(NEW_PORT)
+    assert endpoint_registry_contract.call().findAddressByEndpoint(NEW_PORT) == A
+    assert endpoint_registry_contract.call().findEndpointByAddress(A) == NEW_PORT
+
+    ev_handler.add(txn_hash, 'AddressRegistered', check_address_registered(A, NEW_PORT))
```

Comment:
Fine for now, but we should separate test cases. E.g. the events should be tested in another test. See https://github.com/raiden-network/raiden-contracts/blob/master/raiden_contracts/tests/test_channel_deposit.py#L158 for example.

Reply:
Changed.